acceptance/framework/vault/vault_cluster.go

```diff
@@ -59,13 +59,20 @@ func NewVaultCluster(t *testing.T, ctx environment.TestContext, cfg *config.Test
 	if cfg.EnablePodSecurityPolicies {
 		values["global.psp.enable"] = "true"
 	}
+	if cfg.VaultServerVersion != nil {
+		values["server.image.tag"] = cfg.VaultServerVersion.String()
```

Comment:
If you change VaultServerVersion to a string, this would need to be updated to something like...
```go
if cfg.VaultServerVersion != "" {
    values["server.image.tag"] = cfg.VaultServerVersion
```